kubernetes/loculus/values.yaml

```diff
@@ -1425,7 +1425,6 @@ defaultOrganisms:
     schema:
       image: "https://cdn.who.int/media/images/default-source/mca/mca-covid-19/coronavirus-2.tmb-1920v.jpg?sfvrsn=4dba955c_19"
       organismName: "Test organism (without consensus sequences)"
-      nucleotideSequences: []
```

Comment:
In the schema you say "Defaults to a list with just 'main'." - if so is this problematic?

Reply:
Ah yes, I think you're right.

Reply:
I'll put it back. Thanks for noticing.